app/models/setting.rb

```diff
@@ -287,8 +287,7 @@ def self.set(name, description, default, full_name = nil, value = nil, options =
     if options.has_key? :collection
       SettingsHelper.module_eval do
         define_method("#{name}_collection".to_sym) do
```

Comment:
Do you think we could have a registry on `SettingValueSelection`, that would keep a hash of `[settting_name] => options[:collection]`?
That way we could get rid of this dynamic method definition, in favor of something like `SettingValueSelection.has_collection?(name)` and `SettingValueSelection.collection_for(name)`